RollOfHonour/src/RollOfHonour.Core/Models/Decoration.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace RollOfHonour.Core.Models;
 
-public class Decoration
+public record Decoration
```

Comment:
Nice, records are interesting

Reply:
Don't need to go back and change all the others at this point, but it felt relevant enough to use there. :)